src/DataTypes/LearnerReport.php

```diff
@@ -104,12 +104,12 @@ class LearnerReport {
     /**
      * The UTC date the enrollment was created.
      */
-    protected DateTimeInterface $createdDate;
+    protected ?DateTimeInterface $createdDate;
```

Comment:
At first glance this feels weird how can an enrollment not have a created date... but it is true that the results may not have this filed because the filed may not have been requested. I think some documentation about how not all columns are present unless requested and will default to null if not requested should be added to the class.